x-pack/plugins/graph/public/components/search_bar.tsx

```diff
@@ -9,6 +9,7 @@ import { EuiFlexGroup, EuiFlexItem, EuiButton, EuiButtonEmpty, EuiToolTip } from
 import React, { useState, useEffect } from 'react';
 
 import { i18n } from '@kbn/i18n';
+import { fromKueryExpression, toElasticsearchQuery } from '@kbn/es-query';
```

Comment:
LGTM, just wondering, this little change adds 29kb to Graph?
<img width="443" alt="Bildschirmfoto 2021-12-14 um 10 45 03" src="https://user-images.githubusercontent.com/463851/145973972-a5fbaa73-fa84-45f1-b012-fe8269488097.png">

Reply:
Maybe we should add `@kbn/es-query` to shared deps then? https://github.com/elastic/kibana/blob/992f3ac6918811859a7367e886564ddfcd6095dc/packages/kbn-ui-shared-deps-src/src/index.js#L29

Reply:
this is a good idea, since there seem to be more plugins gaining weight because of this change